hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx

```diff
@@ -21,6 +29,8 @@ const Orders = () => {
         alert(`The order you clicked is: #${orderId}`);
```

Comment:
Can you remove this handleClick function, we don't need to send an alert. It should redirect admins to their team detail page. See my comment below, on how to do that.

Reply:
done